packages/calcite-components/src/components/date-picker-month/date-picker-month.tsx

```diff
@@ -86,26 +141,45 @@ export class DatePickerMonth {
    *
    * @internal
    */
-  @Event({ cancelable: false }) calciteInternalDatePickerSelect: EventEmitter<Date>;
+  @Event({ cancelable: false }) calciteInternalDatePickerDaySelect: EventEmitter<Date>;
 
   /**
    * Fires when user hovers the date.
    *
    * @internal
    */
-  @Event({ cancelable: false }) calciteInternalDatePickerHover: EventEmitter<Date>;
+  @Event({ cancelable: false }) calciteInternalDatePickerDayHover: EventEmitter<Date>;
 
   /**
    * Active date for the user keyboard access.
    *
    * @internal
    */
-  @Event({ cancelable: false }) calciteInternalDatePickerActiveDateChange: EventEmitter<Date>;
+  @Event({ cancelable: false }) calciteInternalDatePickerMonthActiveDateChange: EventEmitter<Date>;
+
+  /**
+   * @internal
+   */
+  @Event({ cancelable: false }) calciteInternalDatePickerMonthMouseOut: EventEmitter<void>;
 
   /**
+   * Emits when user updates month or year using `calcite-date-picker-month-header` component.
+   *
    * @internal
    */
-  @Event({ cancelable: false }) calciteInternalDatePickerMouseOut: EventEmitter<void>;
+  @Event({ cancelable: false }) calciteInternalDatePickerMonthChange: EventEmitter<{
+    date: Date;
+    position: string;
+  }>;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Private State/Props
+  //
+  //--------------------------------------------------------------------------
+
+  /** current focused date */
```

Comment:
Do we need this comment? It seems a bit redundant. Might apply to other comments.